openvpn/middlewares/client/bytescount/middleware_test.go

```diff
@@ -40,6 +40,14 @@ func Test_Start(t *testing.T) {
 	assert.Equal(t, "bytecount 1", mockConnection.LastLine)
 }
 
+func Test_Stop(t *testing.T) {
+	statsRecorder := fakeStatsRecorder{}
+	middleware := NewMiddleware(statsRecorder.record, 1*time.Second)
+	mockConnection := &management.MockConnection{}
+	middleware.Stop(mockConnection)
+	assert.Equal(t, "bytecount 0", mockConnection.LastLine)
```

Comment:
What is the test case here?

Reply:
We had a `Test_Start` a bit upper in the same file. It checks the correctness of the send command on the middleware `Start`. 
I have added the similar test, but for `Stop` case. 
It was added to keep consistency for these tests.

Reply:
Oh, now I got it. Start /stop enables/disables bytecount notifications receiving on requested interval. Yea this looks ok now.